static/src/stylesheets/module/_social.scss

```diff
@@ -176,7 +178,12 @@ category: Common
     background-color: colour(social-pinterest);
 
     // The pinterest icon needs to be centered within the span due to its small width/height
-
+    svg {
```

Comment:
@Pearson82 what do you think? It would save us some code, as there are so many pinterest specific fixes.